src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/DamageableItem.java

```diff
@@ -47,22 +47,25 @@
      *            The {@link ItemStack} to damage
      */
     default void damageItem(@Nonnull Player p, @Nullable ItemStack item) {
-        if (isDamageable() && item != null && item.getType() != Material.AIR && item.getAmount() > 0) {
-            int unbreakingLevel = item.getEnchantmentLevel(Enchantment.DURABILITY);
+        if (isDamageable() && item != null && item.getType() != Material.AIR && item.getAmount() > 0 && item.hasItemMeta()) {
+            ItemMeta meta = item.getItemMeta();
 
-            if (unbreakingLevel > 0 && Math.random() * 100 <= (60 + Math.floorDiv(40, (unbreakingLevel + 1)))) {
-                return;
-            }
+            if (!meta.isUnbreakable()) {
+                Damageable damageable = (Damageable) meta;
```

Comment:
`getItemMeta()` is an expensive call, so please revert this swap around because checking for the unbreaking level first is a rather important performance optimization.